evalml/tests/pipeline_tests/test_time_series_pipeline.py

```diff
@@ -1584,3 +1584,49 @@ def test_time_index_cannot_be_none(time_series_regression_pipeline_class):
                 }
             }
         )
+
+
+def test_time_series_random_forest_transform_all_but_final(
+    ts_data, time_series_regression_pipeline_class
+):
+    X, y = ts_data
```

Comment:
can we drop rows to make this test faster? Think we could just use the minimum amount of rows we need for the forecast horizon.